specs/eip4844/p2p-interface.md

```diff
@@ -117,12 +106,14 @@ Alias `sidecar = signed_blobs_sidecar.message`.
 - _[REJECT]_ the `sidecar.blobs` are all well formatted, i.e. the `BLSFieldElement` in valid range (`x < BLS_MODULUS`).
 - _[REJECT]_ The KZG proof is a correctly encoded compressed BLS G1 Point -- i.e. `bls.KeyValidate(blobs_sidecar.kzg_aggregated_proof)
 - _[REJECT]_ the beacon proposer signature, `signed_blobs_sidecar.signature`, is valid -- i.e.
-```python
+
+```
```

Comment:
```suggestion
```python
```

Reply:
It is a silly workaround for *not* including this code snippet to pyspec. 😅

Alternatively, we can use "```py". Still not perfect though.

Reply:
I changed this code snippet into listed multi-steps: https://github.com/ethereum/consensus-specs/pull/2937/commits/8776c05a2035f0dbbadeaaf6fbb794c54e99713e